### ISSUE TITLE
QuickCreate forms support

### ISSUE THREAD

**Issue by thathi** [DynamicHands/Crm.Specflow#120]
I couldn't find any bindings or code to support QuickCreate FormFill? Can we extend the formdata to support quickcreateforms or another approach?

**Comment by mVermaat** [comment]
Correct, I haven't implemented any quick create yet. It's possible to extend it to use quick create, but not really sure on what basis to decide when to use quick create and when to use the actual form.

**Comment by thathi** [comment]
I will try to come up an option we can tune it based on your review.
